pkg/proxy/error.go

```diff
@@ -83,6 +87,19 @@ func isEOFErr(err error) bool {
 	return false
 }
 
+func isConnEndErr(err error) bool {
+	if err == nil {
+		return false
+	}
+	if strings.Contains(err.Error(), errMsgClosedConn) {
```

Comment:
Do we any anything better than matching substring?

Reply:
modified